backend/back-core/config/index.ts

```
@@ -0,0 +1,113 @@
+  // testnets
+  "22040": "amb-test",
+  "84532": "base-test",
+  "6003100671677645902": "sol-dev",
+};
+
+const networkMappping: Record<string, string> = Object.entries(
+  CHAIN_ID_TO_CHAIN_NAME
+).reduce(
+  (acc: Record<string, string>, [key, value]: [string, string]) => ({
+    ...acc,
+    [value]: key,
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)